packages/pardi/pardi.2.0.4/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/pardi"
+bug-reports: "https://github.com/UnixJunkie/pardi/issues"
+dev-repo: "git+https://github.com/UnixJunkie/pardi.git"
+license: "GPL-1.0-or-later"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {>= "1.11"}
+  "batteries"
+  "dolog" {>= "4.0.0"}
+  "parany" {>= "6.0.0"}
+  "minicli"
```

Comment:
Hasn't minicli also changed it interface name completely in `minicli.5.0.0` or something?

Reply:
I just tried locally and it failed with minicli.4.0.0. I'm fixing that.